latex2sympy.py

```diff
@@ -573,8 +573,17 @@ def convert_binom(binom):
 
 def convert_func(func):
     if func.func_normal():
+        def resolve_args(args):
+            """
+            Turn multiple parameters into a list of parameters
+            """
+            if not hasattr(args, 'expr'):
+                return []
```

Comment:
I'm not quite sure I understand this part, but it seems to be working. `convert_func_arg` checked for this and returned `convert_mp` instead. This seems like it would change the value of `arg` below in `func.L_PAREN` where it could be an empty array now.

Reply:
From what I understand, `convert_func_arg()` should be returning `convert_expr()` because the captured `func_arg` only contains `expr | (expr ',' func_arg)`.

I can also add an if block for the `mp`s just in case.

Reply:
oh I see, if there are parentheses it has to be an `expr`.  I think it's good as-is